src/stm.py

```diff
@@ -30,18 +30,18 @@ def cbf(volume, method = "fft", fps = 200, max_freq = 20):
         A heatmap of dominant frequencies, one at each pixel location.
     """
     if method == "fft":
-        return _cbf_fft(volume, fps, max_freq = max_freq)
+        return _cbf_fft(volume, fps, max_freq = max_freq,sequence=sequence)
```

Comment:
Add proper spacing in accordance with PEP standards:
 - there should be a space after a comma
 - consistent with the rest of the file, there should be spaces on either sides of the equals `=` sign